src/splitInteger.test.js

```diff
@@ -7,13 +7,41 @@ describe(`Function 'splitInteger':`, () => {
     expect(splitInteger).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
-
+  it(`should retur an array`, () => {
+    expect(splitInteger(8, 4))
+      .toBeInstanceOf(Array);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    const expectNum = 3;
 
+    expect(splitInteger(expectNum, 1)).toEqual([3]);
   });
-  
```

Comment:
It's better to add the test with result numbers in the message so the developer could see what is wrong.
For example: `should return an array [4, 4, 4, 5] with num = 17 if parts = 4`